src/qmapshack/mouse/line/IMouseEditLine.cpp

```diff
@@ -312,6 +316,19 @@ void IMouseEditLine::slotTrackRouting() {
   canvas->reportStatus("Routino", QString());
 }
 
+void IMouseEditLine::slotHasFastRouting(bool b) {
+  if (b) {
+    connect(scrOptEditLine->toolAutoRoute, &QPushButton::clicked, this, &IMouseEditLine::slotAutoRouting);
+  } else {
+    disconnect(scrOptEditLine->toolAutoRoute, &QPushButton::clicked, this, &IMouseEditLine::slotAutoRouting);
```

Comment:
do we really need to connect/disconnect the signals while we disable the button?